README.md

```diff
@@ -1,11 +1,97 @@
-# service-ca
-Coming soon...
+# OpenShift Service CA Operator
 
-Controller to mint and manage serving certificates for Kubernetes services.
+This operator runs the following OpenShift service-serving certificate controllers:
+* service-ca controller:
+  * Issues serving certificates based on annotations on services, and provides them via secrets. [See the current OKD documentation for usage.](https://docs.okd.io/latest/dev_guide/secrets.html#service-serving-certificate-secrets)
```

Comment:
How about `Issues a signed serving certificate/key pair to services annotated with <annotation here> via a secret.`? Something more specific about which assets are generated.